src/nashpy/algorithms/lemke_howson_lex.py

```diff
@@ -1,14 +1,15 @@
 """A class for the Lemke Howson algorithm with lexicographical ordering"""
-from itertools import cycle
-
-import numpy as np
+from deprecated import deprecated  # type: ignore
 import numpy.typing as npt
 from typing import Tuple
-from nashpy.integer_pivoting import make_tableau, pivot_tableau_lex
 
-from .lemke_howson import shift_tableau, tableau_to_strategy
+from .lemke_howson import lemke_howson
 
 
+@deprecated(
```

Comment:
We never discussed what to do with the old lex method. I left it in, with a deprecation warning in case its still used by some